lib/util.ts

```diff
@@ -11,3 +15,25 @@ export function determineRunOrder(index: number, concurrency?: number) {
 
   return Math.floor(index / concurrency) + 1;
 }
+
+/**
+ * Hashes the contents of a file or directory. If the argument is a directory,
+ * it is assumed not to contain symlinks that would result in a cyclic tree.
+ *
+ * @param fileOrDir the path to the file or directory that should be hashed.
+ *
+ * @returns a SHA256 hash, base-64 encoded.
+ */
+export function hashFileOrDirectory(fileOrDir: string): string {
```

Comment:
Raise an issue to add this to `FileAsset`

Reply:
awslabs/aws-cdk#1400